packages/fx-core/src/component/configManager/parser.ts

```diff
@@ -39,6 +39,13 @@ function parseRawProjectModel(obj: Record<string, unknown>): Result<RawProjectMo
     return err(new YamlFieldMissingError("version"));
   }
 
+  if ("sampleTag" in obj) {
+    if (typeof obj["sampleTag"] !== "string") {
+      return err(new YamlFieldTypeError("sampleTag", "string"));
```

Comment:
@chagong Since sampleTag is optional and it doesn't block anything when the value is invalid, I personally recommend ignore this schema error instead of throw error.
cc @1yefuwang1

Reply:
This is good observation which I missed. 
The validation error is dismissed in https://github.com/OfficeDev/TeamsFx/pull/9533